Library/Formula/caf.rb

```diff
@@ -0,0 +1,52 @@
+class Caf < Formula
+  homepage "http://actor-framework.org"
+  head "https://github.com/actor-framework/actor-framework.git"
+
+  stable do
+    url "https://github.com/actor-framework/actor-framework/archive/0.12.0.tar.gz"
+    sha256 "294fb00508c166595d479e9af2fae28f2d370ed164cd1de079ae247acc916801"
+  end
+
+  devel do
+    url "https://github.com/actor-framework/actor-framework.git",
+      :branch => "develop"
```

Comment:
Is this not a ` head do ` rather than a ` devel do ` since it points to a ` .git `?